experimental/traffic-portal/src/app/core/cache-groups/cache-group-details/cache-group-details.component.html

```diff
@@ -56,11 +56,11 @@
 			<div class="pair">
 				<mat-form-field>
 					<mat-label>Latitude</mat-label>
-					<input type="number" matInput min="-90" max="90" step="0.001" name="latitude" [(ngModel)]="cacheGroup.latitude" required/>
+					<input type="number" matInput min="-90" max="90" step="0.0000001" name="latitude" [(ngModel)]="cacheGroup.latitude" required/>
```

Comment:
I think the step should have much higher of a step. According to [this](https://gis.stackexchange.com/questions/8650/measuring-accuracy-of-latitude-and-longitude/8674#8674) (You can just skip to the list at the bottom of the answer), the first decimal seems like a much better step. TPv1 uses integer steps but I think that's not enough precision.